typings/xterm.d.ts

```diff
@@ -638,7 +638,7 @@ declare module 'xterm' {
     /**
      * Get the terminal options
```

Comment:
```suggestion
     * Gets or sets the terminal options. This supports setting multiple options.
     *
     * @example Get a single option
     * ```typescript
     * console.log(terminal.options.fontSize);
     * ```
     *
     * @example Set a single option
     * ```typescript
     * terminal.options.fontSize = 12;
     * ```
     *
     * @example Set multiple options
     * ```typescript
     * terminal.options = {
     *   fontSize: 12,
     *   fontFamily: 'Arial',
     * ;
     * ```
     */
```